modules/apps/list-type/list-type-service/src/main/java/com/liferay/list/type/service/impl/ListTypeDefinitionServiceImpl.java

```diff
@@ -124,14 +126,15 @@ public int getListTypeDefinitionsCount() {
 	@Override
 	public ListTypeDefinition updateListTypeDefinition(
 			String externalReferenceCode, long listTypeDefinitionId,
-			Map<Locale, String> nameMap)
+			List<ListTypeEntry> listTypeEntries, Map<Locale, String> nameMap)
```

Comment:
I think we should order the parameters by following the order from service.xml. Since ListTypeDefinition does not contain the listTypeEntries, so I believe that this parameter should come at last.